x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/policy_tabs.tsx

```diff
@@ -65,7 +65,9 @@ export const PolicyTabs = React.memo(() => {
   });
 
   const canSeeHostIsolationExceptions =
-    privileges.canIsolateHost || allPolicyHostIsolationExceptionsListRequest.data?.total !== 0;
+    privileges.canIsolateHost ||
+    (typeof allPolicyHostIsolationExceptionsListRequest.data !== 'undefined' &&
```

Comment:
nit: You could avoid the typeof by using `!!allPolicyHostIsolationExceptionsListRequest.data`

Reply:
You could also check if `!allPolicyHostIsolationExceptionsListRequest.isLoading` before check if there is data

Reply:
Yeah, those are also fine to check. Additionally one will have to still check if there is `data` as `isLoading` or `isFetched` do not tell TS about whether data exists as data can be `undefined`.

This is neater and does all that.

I'll change this to the following if you still think we should use `isLoading` or `isFetched`.
```
(allPolicyHostIsolationExceptionsListRequest.isFetched &&
      allPolicyHostIsolationExceptionsListRequest.data?.total !== 0);
```

Reply:
done